build/charts/antrea/README.md

```diff
@@ -90,6 +90,9 @@ Kubernetes: `>= 1.16.0-0`
 | multicluster.enablePodToPodConnectivity | bool | `false` | Enable Multi-cluster Pod to Pod connectivity. |
 | multicluster.enableStretchedNetworkPolicy | bool | `false` | Enable Multi-cluster NetworkPolicy. Multi-cluster Gateway must be enabled to enable StretchedNetworkPolicy. |
 | multicluster.namespace | string | `""` | The Namespace where Antrea Multi-cluster Controller is running. The default is antrea-agent's Namespace. |
+| multicluster.trafficEncryptionMode | string | `"none"` | Determines how cross-cluster traffic is encrypted. It has the following options: - none (default):  Cross-cluster traffic will not be encrypted. - wireGuard:       Enable WireGuard for tunnel traffic encryption. |
+| multicluster.wireGuard | object | `{"port":51821}` | WireGuard tunnel configuration for cross-cluster traffic. |
+| multicluster.wireGuard.port | int | `51821` | WireGuard tunnel port for cross-cluster traffic. |
```

Comment:
This is the first one generating duplicate items for a config, I confirmed with @antoninbas and we think only `multicluster.wireGuard.port` should be here. The other item will be deleted automatically if you change the leading charactors of the comment of `wireGuard` in values.yaml from `# --` to `#`

Reply:
Changed  `# --` to `#` and re-generated the doc.